tools/run_tests/dockerize/build_docker_and_run_tests.sh

```diff
@@ -82,6 +82,7 @@ docker run \
   -e "BUILD_URL=$BUILD_URL" \
   -e "JOB_BASE_NAME=$JOB_BASE_NAME" \
   -i $TTY_FLAG \
+  --sysctl net.ipv6.conf.all.disable_ipv6=0 \
```

Comment:
When I tested without this (https://github.com/matt-kwong/grpc/commit/d3fe9435769441908dcf13583c40ea672d111d70), the build stalled on trying to assign an IPv6 address. It might be the case that 
```
sudo sed -i s/DOCKER_OPTS=/DOCKER_OPTS=\"--ipv6\"/g /etc/init.d/docker
sudo /etc/init.d/docker restart
```
isn't necessary. I'll test if that's the case.